web/client/plugins/Dashboard.jsx

```diff
@@ -125,13 +125,30 @@ const WidgetsView = compose(
  * @prop {object} cfg.cols Number of columns in this layout. default { lg: 6, md: 6, sm: 4, xs: 2, xxs: 1 }
  * @prop {object} cfg.minLayoutWidth minimum size of the layout, below this size the widgets are listed in a single column
  * for more info about rowHeight and cols, see https://github.com/STRML/react-grid-layout#grid-layout-props
+ * @prop {object} cfg.widgetOpts widget specific options
+ * @example
+ * {
+ *   "name": "Dashboard",
+ *   "cfg": {
+ *      "rowHeight": 150,
+ *      "cols": { lg: 6, md: 6, sm: 4, xs: 2, xxs: 1 },
+ *      "widgetOpts": {
+ *          "table": {
```

Comment:
can you list somewhere above the list of entries that can be used ? table, counter etc.